### DISCUSSION TITLE
Is it possible to find the project boards an issue is associated with using the REST API?

### DISCUSSION THREAD

**Discussion by svpr3m0** [community/community#24740] [Apps, API and Webhooks]
<p>I’m developing a Github webhook that gets notifications on issue updates.</p>
<p>These issues are associated to one of more <a href="https://docs.github.com/en/github/managing-your-work-on-github/about-project-boards" rel="nofollow noopener">Github Project Boards</a></p>
<p>The body payload shows this information related to the updated issue:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/0/0d08b1d42d03980163866ffb0cff0021225d445d.jpeg" data-download-href="/uploads/short-url/1RiPMUoopuDpE2Gm7n0EnX1qbmZ.jpeg?dl=1" title="image" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/67064183/181086519-924dacae-d6b3-45ae-ac07-5a3412c7d395.jpeg" alt="image" data-base62-sha1="1RiPMUoopuDpE2Gm7n0EnX1qbmZ" width="182" height="500" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/0/0d08b1d42d03980163866ffb0cff0021225d445d_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">233×640 58 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Nothing about issue association with project boards. Is there a way to get this information?</p>

**Comment by francisfuzz** [comment, marked as answer]
<p>:wave: hello there <a class="mention" href="/svpr3m0">@svpr3m0</a>, and  welcome to the GitHub Support Community!</p>
<p>Making a request to the REST API for <a href="https://docs.github.com/en/rest/reference/issues#get-an-issue">getting an issue</a> will not include the project that an issue is a part of.</p>
<p>Instead, you can go through the list of cards in the project (using the <a href="https://docs.github.com/en/rest/reference/projects">Projects API</a>, specifically the <a href="https://docs.github.com/en/rest/reference/projects#list-project-cards">list project cards</a> endpoint).</p>
<p>When you fetch a card, the <code>content_url</code> field will point to an issue if the card was created from an issue. I hope this helps!</p>

**Comment by svpr3m0** [comment]
<p>Hi <a class="mention" href="/francisfuzz">@francisfuzz</a>, thank you for your answer. What you suggest could be a valid workaround though I guess it’s not really optimal since I would have to go through every project card to match the issue which triggered the webhook.</p>
<p>Anyway that’s the way the API is hehe. Thanks again!</p>

**Comment by jongio** [comment]
<p>I needed the ability to get all issues assigned to a project and found this.</p>
<p>Is “get issues in project” going to be added to the REST API backlog?</p>

**Comment by foodelix-admin** [comment]
It would be extremely useful to get the issue related to a project card, at the moment I couldn't find a way to do it other than iterating on all project cards
